destinations/data/jaeger.yaml

```diff
@@ -20,4 +20,28 @@ spec:
       componentProps:
         type: text
         required: true
+    - name: JAEGER_CERT_PEM
+      displayName: Client Certificate
+      componentType: textarea
+      componentProps:
+        type: password
+        required: false
+        placeholder: '-----BEGIN CERTIFICATE-----'
+        tooltip: 'When using TLS, the client certificate is required.'
+    - name: JAEGER_KEY_PEM
+      displayName: Client Key
+      componentType: textarea
+      componentProps:
+        type: password
+        required: false
+        placeholder: '-----BEGIN PRIVATE KEY-----'
+        tooltip: 'When using TLS, the client key is required.'
```

Comment:
we don't need the server to authenticate clients. These should be removed I think